test/support/test_utils.ex

```diff
@@ -11,6 +11,10 @@ defmodule ETS.TestUtils do
     wait_until_dead(pid)
   end
 
+  def otp25? do
+    :erlang.system_info(:otp_release) |> to_string() |> String.starts_with?("25")
```

Comment:
I just realized that this will break when OTP 26+ is released. Could you update this to parse the string as an integer and verify >= 25?

Reply:
Done!